ExpressionEngine 7.4.1

matthewjohns0n opened the pull request:
changelog: https://github.com/ExpressionEngine/ExpressionEngine-User-Guide/pull/811

matthewjohns0n committed:
Remove unnecessary use statement

matthewjohns0n committed:
Fix issue with the filesystem loader

matthewjohns0n committed:
Fix namespace of exception handler in the backup:database command

Resolves #3886

bryannielsen committed:
Support color picker swatches in channel forms

intoeetive committed:
Resolved #3894 where PHP error could be shown when displaying CAPTCHA using PHP 8.1

intoeetive committed:
Resolved #3913 where PHP deprecation error could be shown on entry manager page when entry author did not exist

intoeetive committed:
version 7.4.1

matthewjohns0n committed:
FIxes category groups no results tag in channel edit. Fixes #4048

intoeetive committed:
set DP version for pre-release

intoeetive committed:
fix the updater file name

intoeetive committed:
fix setting version for 1-click updater test

intoeetive committed:
do what you have to do

intoeetive committed:
Resolved #4050 where error was shown in Channel Form when multiple category groups were assigned

intoeetive committed:
Resolved issue when category group settings were not retained after editing from Channel page

intoeetive committed:
Resolved issue where `{categories}` variable pair was not populated in Channel Form

intoeetive committed:
Merge branch '7.dev' into feature/7.x/colorpicker-swatches-channel-form

intoeetive committed:
Merge pull request #3882 from ExpressionEngine/bug/7.x/error-with-update-prepare-command

Resolved #3883 where update:prepare command would throw errors

intoeetive committed:
Merge pull request #3900 from ExpressionEngine/feature/7.x/colorpicker-swatches-channel-form

Support color picker swatches in channel forms

matthewjohns0n committed:
Merge pull request #4044 from ExpressionEngine/release/ee-7.4.1

version 7.4.1

intoeetive commented:
The failed test is due to files being uploaded in different order... the fix for the test is coming in 7.4.2

intoeetive committed:
Resolved issue where only one category group was shown in Categories filter in Entry manager if the channel is selected

Resolved issue where Categories filter was empty if no channel is selected in Entry Manager

intoeetive committed:
remove second parameter
